contents/basepages/aboutus/index.en.mdx

```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
+In other words, there was a method of advertising, publicity, and sales that was optimal for the size of the store and the size of the trade area.
+
+It's called DRM (Direct Response Marketing).
+Moreover, it is similar to the way of thinking and way of thinking that was done in traditional Japanese business.
+
+Do you know Drucker, who is famous for "Moshi Dora"?
+It is said that Echigoya in the Edo period started the world's oldest marketing.
+
+So, if you are Japanese, the content of my Google Maps attracting customers using DRM and DRM is
+It's all about the things that make sense honestly.
+
+However, when applying the Internet,
+Internet and computer knowledge are required.
+
+It's not easy for anyone to use.
+
+### Meet Google Maps, easy to use on your smartphone
+
+However, it is ideal for small local shops,
+Anyone can use it easily with a smartphone,
+And I found out about free Google business profiles.
+Google business profile
+It is a tool to manage the information of the shop placed on Google Maps.
+
+In fact, at that time and now, in Japan, it increases the search ranking of Google Maps
+Even if there is information on "MEO (map engine optimization) measures",
+There is not much information that actually leads to the store from there.
+
+Therefore, I decided to obtain information in English from overseas at that time.
+It turns out that the word "MEO" is unique to Japan and was not used overseas.
+
+Instead, Google officially used "Local SEO"
+(Even if there is a search engine,
+There is no such thing as a map engine. )
+
+Local SEO is not local SEO.
+It's a local search engine optimization.
+Again, this is confusing, but the English word "local" doesn't seem to mean rural or country.
+Ultimately, it is a pair of words for global (whole), and it seems that regional and local are more appropriate.
+
+Then I started researching local SEO and started attracting customers on Google Maps.
+
+
+  Because I want to spread the way to let you know
+* If you make a mistake in notifying, not only will customers not come,
+  because i lose money
+  ## People who have practiced Google Maps attracting customers...
+### Single-owner nail salon<br/> For A.O.
+
+#### Before starting to attract customers to Google Maps
+
+<ImageBox alt="Testimonial from Nail Salon" float="right"
+       image={1} bgColor="transparent"
+       style={{borderWidth: "1px"}} />
+
+* No new acquisitions from 2 years ago
+* The number of existing customers has decreased sharply due to the corona crisis
+* I often feel depressed, and with my old mother in my hometown
+  I was beginning to think about living a simple and leisurely life.
+
+#### When you start attracting customers to Google Maps
+
+* Newly acquired for the first time in two years in just one week
+* Constant new acquisition just by uploading photos
+* Now he even wants to open a nail salon in his hometown.
+  I became positive and cheered up
```

Comment:
Please ignore the line break and translate the line.

Reply:
DONE